config/routes.rb

```diff
@@ -30,7 +30,10 @@
         get "record-template", on: :collection
       end
 
-      get "/consent/confirm", to: "consent_responses#confirm", on: :member
+      resource :consent_responses, path: "consent" do
+        post "confirm"
+        put "record"
```

Comment:
I get why this is `record` and not `create` ... this action will only update `recorded_at`, but it still feels odd.